frontend/src/store/ConfigStore.ts

```diff
@@ -82,7 +82,7 @@ const configureStore = (initialState: KialiAppState): Store<KialiAppState, Kiali
 // Setup the initial state of the Redux store with defaults
 // (instead of having things be undefined until they are populated by query)
 // Redux 4.0 actually required this
-const initialStore: KialiAppState = {
+export const initialStore: KialiAppState = {
```

Comment:
Does this need the export?

Reply:
Nope this was leftover from testing but I ended up not using this. I'll remove.